src/db/collections/outputs/indexer/mod.rs

```diff
@@ -378,7 +378,9 @@ impl OutputCollection {
 
         self.create_index(
             IndexModel::builder()
-                .keys(doc! { "metadata.spent_metadata.spent.milestone_index": -1 })
+                .keys(
+                    doc! { "metadata.spent_metadata.spent.milestone_index": -1, "metadata.booked.milestone_index": -1, "details.address": 1 },
+                )
                 .options(
                     IndexOptions::builder()
                         .name("output_spent_milestone_index".to_string())
```

Comment:
I think we should rename this index, so that people don't have to resync the database / clear the indexes.

Reply:
Renamed the index, and got rid of the `details.address` field (which we don't really need any more). Leaving this discussion unresolved for reference, feel free to close it though.